python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -874,7 +888,18 @@ def __name__(self, value):
         setattr(self._fsproxy_slow, "__name__", value)
 
 
-def _fast_slow_function_call(func: Callable, /, *args, **kwargs) -> Any:
+def _assert_fast_slow_eq(left, right, **kwargs):
+    assert_func = kwargs.get("assert_func", assert_eq)
+    if _is_final_type(type(left)) or (type(left) in NUMPY_TYPES):
+        assert_func(left, right)
+
+
+def _fast_slow_function_call(
+    func: Callable,
+    mode_pandas_debugging: bool | None = None,
```

Comment:
*TODO*: I think you need to plumb this through the uses of `_fast_slow_function_call` in `_wrappers/pandas.py`